vowpalwabbit/global_data.cc

```diff
@@ -437,4 +438,45 @@ vw::vw()
   sd->multiclass_log_loss = 0;
   sd->holdout_multiclass_log_loss = 0;
 }
-IGNORE_DEPRECATED_USAGE_END
\ No newline at end of file
+IGNORE_DEPRECATED_USAGE_END
+
+vw::~vw()
```

Comment:
did you give this a pass to check if we are missing something that got added to the class but not to the old parse args destructor logic? feel that causes room for mismatch

Reply:
Yeah I did a pass and deprecated an unused variable. `searchstr` seems fishy to me but don't want to deal with it in this PR